examples/events.py

```diff
@@ -1,217 +1,123 @@
+    print(f"{new_received} of {new_received.clan} just received {final_received_troops} troops.")
+
+
+@client.event
+@coc.ClanEvents.member_join(clan_tags, retry_interval=20)
+async def on_clan_member_join(member, clan):
+    print(f"{member.name} has joined {clan.name}")
+
+
+@client.event
+@coc.ClanEvents.member_leave(clan_tags, retry_interval=20)
+async def on_clan_member_leave(member, clan):
+    print(f"{member.name} has left {clan.name}")
+
+
+@client.event
+@coc.ClanEvents.points(clan_tags, retry_interval=10)
+async def on_clan_trophy_change(old, new):
+    print(f"{new.name} total trophies changed from {old.points} to {new.points}")
+
 
 @client.event
-@coc.ClanEvents.member_donations_change(c_tags, retry_interval=0)
-async def on_member_dono_change(old, new):
-    # print(old.donations, new.donations)
-    ...
+@coc.ClanEvents.member_versus_trophies(clan_tags, retry_interval=10)
+async def on_clan_versus_change(old, new):
+    print(f"{new.name} total versus trophies changed from {old.versus_trophies} to {new.versus_trophies}")
 
 
-@coc.WarEvents.war_attack()
-async def test():
-    pass
+"""War Events"""
 
 
-@coc.WarEvents.war_attack()
-async def test():
-    pass
+@client.event
+@coc.WarEvents.war_attack(clan_tags, retry_interval=30)
+async def current_war_stats(attack, war):
+    print(f"Attack number {attack.order}\n({attack.attacker.map_position}).{attack.attacker} of {attack.attacker.clan} attacked ({attack.defender.map_position}).{attack.defender} of {attack.defender.clan}")
 
 
-@coc.WarEvents.war_attack()
-async def callable():
-    pass
+"""Player Events"""
 
 
 @client.event
-@coc.WarEvents.war_attack(c_tags, retry_interval=30)
-async def change2(attack, new):
-    # print(attack.attacker, attack.order, max(a.order for a in new.attacks))
-    # log.info("ran war")
-    ...
+@coc.PlayerEvents.donations(tags=player_tags)
+async def on_player_donation(old_donation, new_donation):
+    final_donated_troops = new_donation.donations - old_donation.donations
+    print(f"{new_donation} of {new_donation.clan} just donated {final_donated_troops} troops.")
 
 
 @client.event
-@coc.PlayerEvents.trophies_change(retry_interval=0)
+@coc.PlayerEvents.received(tags=player_tags)
+async def on_player_donation_receive(new_received, old_received):
+    final_received_troops = old_received.received - new_received.received
+    print(f"{new_received} of {new_received.clan} just received {final_received_troops} troops.")
+
+
+@client.event
+@coc.PlayerEvents.trophies(tags=player_tags)
 async def on_player_trophy_change(old, new):
-    # print(old.trophies, new.trophies)
-    ...
+    print(f"{new} trophies changed from {old.trophies} to {new.trophies}")
+
+
+@client.event
+@coc.PlayerEvents.versus_trophies(tags=player_tags)
+async def on_player_versus_trophy_change(old, new):
+    print(f"{new} versus trophies changed from {old.trophies} to {new.trophies}")
 
 
+"""Client Events"""
```

Comment:
Maybe a brief (1-2 line) description of what client events are? (they're events from coc.py that provide additional information about the status of the API and events loops, such as maintenance, when IP changes, etc.)